### ISSUE TITLE
Can't connect to Bitcoin Core Daemon

### ISSUE THREAD

**Issue by crusher72** [bonesoul/CoiniumServ#896]
[Error] [ShareManager] [Bitcoin] Error getting account for pool central wallet address: 16rqHVnZcrzLGtjA2KuPyr4TRGTZD67UCG - An unknown exception occured while reading json response

Should I try a different wallet? Or do I need to add code for BIP-145?

**Comment by bonesoul** [comment]
Version of core? 0.16? If so not supported yet.

**Comment by crusher72** [comment]
v0.14.1

I have an older version of CS (0.2.4.32298) with modifications for Segwit working with LTC.

But It won't read the json response correctly for Bitcoin. If I update to the newest version of CS, will it work?

I saw a different vegabond version, but I want to stick with CSredis for now if I can. Plus my version of CS is very stable. If I could, I would like to just update the rules where it applies for Segwit acceptance of Bitcoin blocks. If this is un-necessary let me know what the best direction I should go in to get this to work properly.

**Comment by crusher72** [comment]
I am also willing to send you any work that is done, to submit a PR to help this project, I don't want to just take, but I will give back if I get some help on what to do.

**Comment by bonesoul** [comment]
Feel fee to PR, I'll be merging in.

**Comment by crusher72** [comment]
For sure, I will and that will be great. But any suggestions anyone for the best direction to go in?

Litecoins requirement which I tested on main net was easy, adding accept segwit to rules, 3 lines of code basically. But what is needed for Bitcoin? I've seen the wiki on BIP-143 but its examples are not that great.

**Comment by bonesoul** [comment]
I've not checked bitcoin changes yet myself. Other then that Bitcoin 0.16 requires more changes as i guess they are dropping some rpc apis; https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes.md

**Comment by crusher72** [comment]
Yeah, the low level RPC calls are depreciated. I will look into what is available, if you see more of what to modify point me there and I'll do the work.

**Comment by bonesoul** [comment]
we have to replace those old rpc calls; https://github.com/bonesoul/CoiniumServ/blob/develop/src/CoiniumServ/Daemon/DaemonClient.cs#L256

**Comment by naheedakhtar** [comment]
> Version of core? 0.16? If so not supported yet.

Which is the maximum supported version...

**Comment by bonesoul** [comment]
0.15 yet. 0.16 is still not released.

**Comment by gblass** [comment]
would this also be the reason why this wouldnt work with mooncoin wallet on Windows.

**Comment by bonesoul** [comment]
nope.

**Comment by naheedakhtar** [comment]
Thanks for amazing software...i was able to install and run properly...however I had to use the fork by [vagabondan](https://github.com/vagabondan/CoiniumServ); also request to add the functionality for supporting equihash algorithm...I am willing to support in any way i can.

**Comment by gblass** [comment]
I can't get the mooncoin-qt on windows to speak to the server.  I have tried multiple rpc ports.  Any ideas.

**Comment by mja1175** [comment]
Anyone now of any version of Coiniumserv that runs with the latest bitcoin?
